x-pack/plugin/ml-package-loader/src/test/java/org/elasticsearch/xpack/ml/packageloader/action/TransportLoadTrainedModelPackageTests.java

```diff
@@ -123,18 +134,24 @@ public void testDoesNotCallListenerWhenNotWaitingForCompletion() {
         );
     }
 
-    private void assertUploadCallsOnFailure(Exception exception, String message, RestStatus status) throws URISyntaxException, IOException {
+    private void assertUploadCallsOnFailure(Exception exception, String message, RestStatus status, Level level) throws URISyntaxException,
+        IOException {
         var esStatusException = new ElasticsearchStatusException(message, status, exception);
 
-        assertNotificationAndOnFailure(exception, esStatusException, message);
+        assertNotificationAndOnFailure(exception, esStatusException, message, level);
     }
 
-    private void assertUploadCallsOnFailure(ElasticsearchStatusException exception, String message) throws URISyntaxException, IOException {
-        assertNotificationAndOnFailure(exception, exception, message);
+    private void assertUploadCallsOnFailure(ElasticsearchException exception, String message, Level level) throws URISyntaxException,
+        IOException {
+        assertNotificationAndOnFailure(exception, exception, message, level);
     }
 
-    private void assertNotificationAndOnFailure(Exception thrownException, ElasticsearchStatusException onFailureException, String message)
-        throws URISyntaxException, IOException {
+    private void assertNotificationAndOnFailure(
+        Exception thrownException,
+        ElasticsearchException onFailureException,
```

Comment:
`ElasticsearchException` has the `status()` used in the test here. 

Changing `ElasticsearchStatusException` to `ElasticsearchException` simplified the test I added